torch/_torch_docs.py

```diff
@@ -736,11 +736,16 @@ def parse_kwargs(desc):
 
 add_docstr(torch.cat,
            r"""
-cat(seq, dim=0, out=None) -> Tensor
+cat(seq, dim=0, pad_value=None, out=None) -> Tensor
 
 Concatenates the given sequence of :attr:`seq` tensors in the given dimension.
-All tensors must either have the same shape (except in the concatenating
-dimension) or be empty.
+If pad_value is set to None, all tensors must have the same shape,
+except in the dimension corresponding to :attr:`dim`.
+If pad_value is set to a float, The size of the resulting tensor is the size of the
```

Comment:
what's special about pad_value being a float?  Isn't an integer ok?  PyTorch 0-dimensional array?  In the `fill_` docs we use "scalar" which seems fine here.